### ISSUE TITLE
Pagination without total number of items

### ISSUE THREAD

**Issue by edewit** [patternfly/patternfly-react#4704]
In some scenarios it's to expensive to get the total number of items. It would be nice if we could still use the Pagination component without that.

**Comment by tlabaj** [comment]
@edewit  is this still an issue for you?

**Comment by edewit** [comment]
Yeah, my "solution" is more of a workaround also it seems that this "toggleTemplate" is ignored when the on mobile
![image](https://user-images.githubusercontent.com/51133/97540549-17e37880-19c4-11eb-8fc3-a03ddab549a0.png)
same screen but then "mobile":
![image](https://user-images.githubusercontent.com/51133/97540639-33e71a00-19c4-11eb-8f97-1c313995064a.png)

**Comment by edewit** [comment]
Also I'm having to dynamically set the `itemCount`, so that the next button appears. I only know there is a next so if there is I make `itemCount` = `perPage` + 1

**Comment by tlabaj** [comment]
cc @mcarrano

**Comment by mcarrano** [comment]
Transferring to design
